src/client/kotlin/tech/thatgravyboat/skycubed/features/overlays/PickUpLog.kt

```diff
@@ -0,0 +1,124 @@
+package tech.thatgravyboat.skycubed.features.overlays
+
+import net.minecraft.client.gui.GuiGraphics
+import net.minecraft.world.item.ItemStack
+import tech.thatgravyboat.skyblockapi.api.events.base.Subscription
+import tech.thatgravyboat.skyblockapi.api.events.base.predicates.TimePassed
+import tech.thatgravyboat.skyblockapi.api.events.hypixel.ServerChangeEvent
+import tech.thatgravyboat.skyblockapi.api.events.screen.PlayerInventoryChangeEvent
+import tech.thatgravyboat.skyblockapi.api.events.time.TickEvent
+import tech.thatgravyboat.skyblockapi.helpers.McClient
+import tech.thatgravyboat.skyblockapi.utils.text.Text
+import tech.thatgravyboat.skycubed.api.displays.Display
+import tech.thatgravyboat.skycubed.api.displays.Displays
+import tech.thatgravyboat.skycubed.api.displays.toColumn
+import tech.thatgravyboat.skycubed.api.displays.toRow
+import tech.thatgravyboat.skycubed.api.overlays.Overlay
+import tech.thatgravyboat.skycubed.config.PickUpLogConfig
+
+object PickUpLog : Overlay {
+    override val name = Text.of("Item Pick Up Log")
+    override val position = PickUpLogConfig.position
+    override val bounds get() = dimension
+
+    private var dimension = 0 to 0
+    private var display: Display? = null
+
+    private val addedItems = mutableListOf<PickupItem>()
+    private val removedItems = mutableListOf<PickupItem>()
+    private var lastInventory = mutableMapOf<Int, ItemStack>()
+
+    private var lastWorldSwitchTime: Long? = null
+
+    @Subscription
+    fun onInvChange(event: PlayerInventoryChangeEvent) {
+        if (event.slot == 8) return
+        if (McClient.self.screen != null) return
+
+        val newStack = event.item
+        val oldStack = lastInventory[event.slot]
+
+        val newCount = newStack.count
+        val oldCount = oldStack?.count ?: 0
+        val diff = newCount - oldCount
+
+        if (diff != 0) {
+            val (targetList, stackToUse) = if (diff > 0) {
+                addedItems to newStack
+            } else {
+                removedItems to (oldStack ?: newStack)
+            }
+
+            val existingItem = targetList.find { it.stack.item == stackToUse.item }
```

Comment:
use isSameItem from sbapi because rn if I pick up for example an enchanted wheat and a normal wheat theyd be considered the same item